### PR TITLE
Fix #72 and #71, Upgrades to PyQt 5 and sequence count fix

### LINKED ISSUE #71: Sequence count likely not coming from header
**Describe the bug**
Looks like sequence count as displayed on the telemetry subsystem pages is not the sequence count from the header

**To Reproduce**
1. Start cFE, start cFS-GroundStation, open cmd and tlm pages
1. Send enable tlm command to 127.0.0.1
1. Wait for packet counter to increment (> 1)
1. Open ES subsystem page, sequence count will start at 1 and increment (should start higher since packets have been received)
1. Close ES subsystem page, reopen and sequence count will start at 1 again

**Expected behavior**
Sequence count should be coming from the packet header, wouldn't restart every time the page opens.

**Code snips**
Haven't investigated yet

**System observed on:**
 - Hardware; cFS Dev Server 3
 - OS: Ubuntu 18.04
 - Versions: Current bundle

**Additional context**
Found during #66 testing

**Reporter Info**
Jacob Hageman - NASA/GSFC

### LINKED ISSUE #72: Update PyQt4 to PyQt5
**Describe the solution you'd like**
All usage of PyQt4 should be updated to PyQt5

**Additional context**
PyQt4 is extremely old at this point, and unless there is a specific need, it should not be used

Note: This fix will include code cleanup/updates as necessary, as well as fix #71 and #79 

**Requester Info**
Leor Bleier, NASA GSFC\Code 582

### PR DESCRIPTION
**Pull request by lbleier-GSFC**
**Describe the contribution**
This is a major change, upgrading PyQt4 to PyQt5 and includes a lot of cleanup/refactoring, and changes to the GUI itself

Fix #71 
Fix #72

**Testing performed**
Steps taken to test the contribution:
1. Make changes
2. Run software to see that it works

**Additional context**
PLEASE NOTE: All `.ui` files as well as any `.py` files auto-generated by `pyuic5` should be ignored during review, as these are related to the GUI itself

**Contributor Info - All information REQUIRED for consideration of pull request**
Leor Bleier, NASA GSFC\Code 582

### CODE REVIEW

**Comment by skliper** [comment]
Marking as not ready until the #83 commits get sorted out (diverged from IC), #81 gets removed (not ready commit), and history gets squashed.

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
 *.pyc
 cmdUtil
-.DS_Store
\ No newline at end of file
+.DS_Store
+Subsystems/cmdGui/CHeaderParser.py
```
Comment (astrogeco) [inline .gitignore:4]: Check why these are here.

File: Subsystems/cmdGui/CommandSystem.py [hunk 2]
```diff
@@ -17,227 +17,104 @@
...
+        pickle_file = f'{ROOTDIR}/ParameterFiles/{quickParam[idx]}'
         try:
-            with open(pickle_file,'rb') as pickle_obj:
-                dataTypesOrig, paramNames, paramLen, paramDesc, dataTypesNew, stringLen = pickle.load(pickle_obj)
-                if len(paramNames) > 0: # if has parameters
-                   return True
-                else:
-                   return False
+            with open(pickle_file, 'rb') as pickle_obj:
+                _, paramNames, _, _, _, _ = pickle.load(pickle_obj)
```
Comment (CDKnightNASA) [inline Subsystems/cmdGui/CommandSystem.py:79]: minor quibble, I'd rather this be "paramNames = pickle.load(pickle_obj)[1]"

File: Subsystems/tlmGUI/EventMessage.py [hunk 3]
```diff
@@ -183,42 +189,39 @@ def usage():
...
             subscription = arg
 
-    if len(subscription) == 0:
+    if not subscription or len(subscription.split('.')) < 3:
         subscription = "GroundSystem"
 
-    arr = subscription.split('.')
-    if len(arr) < 3:
-        subscription = 'GroundSystem'
+    # arr = subscription.split('.')
```
Comment (astrogeco) [inline Subsystems/tlmGUI/EventMessage.py:208]: Clean  up commented code

File: Subsystems/cmdGui/CommandSystem.py (hunk 2 above)
Comment (lbleier-GSFC) [reply Subsystems/cmdGui/CommandSystem.py:79]: Ah, yes. I see what you mean. I can do that

**Comment by skliper** [comment]
CCB 20200513 - Approved with cleanup needed

**Comment by astrogeco** [comment]
@lbleier-GSFC can you take a look at resolving the comments and conflicts?

**Comment by lbleier-GSFC** [comment]
> @lbleier-GSFC can you take a look at resolving the comments and conflicts?

Should be ok now

**Comment by astrogeco** [comment]
@lbleier-GSFC Thanks! Can you rebase and squash commits

**Comment by lbleier-GSFC** [comment]
> @lbleier-GSFC Thanks! Can you rebase and squash commits

Please verify this looks ok

**Comment by skliper** [comment]
@lbleier-GSFC still needs a rebase.  Don't merge ic into your work, rebase your work on ic.

**Comment by skliper** [comment]
@astrogeco fixed!

**Pull request merged**